examples/cpp/CMakeLists_Standalone.txt

```diff
@@ -3,6 +3,8 @@ project(example)
 
 include(ExternalProject)
 
+option(NAVIGATOR_LIB_DIR "Path to Navigator assets (skips download)" "")
+
 if(NOT DEFINED NAVIGATOR_VERSION)
```

Comment:
Missing if(DEFINED NAVIGATOR_LIB_DIR) to avoid status message related to navigator lib version and setting ZIP_URL, it should say that is using local version